### DISCUSSION TITLE
Help with modelling quantity discounts across different products

### DISCUSSION THREAD

**Discussion by Krislunde** [swellstores/community#406] [Help] [answered]
Hey!

So we're currently in the process of building out the store for one of our clients, and they have requested the following pricing structure. They sell spirits and liqueurs, and want people to be able to match a certain amount of bottles and get discounts within each category of either spirits or liqueurs:

1 x Bottle = $41.95
3 x Bottles = $39.95
6 x Bottles = $37.95
12 x Bottles =$35.95

Ideally, these can be mixed and matched across bottles. 

(Liqueurs will need to be as above but $10 less)

How would I go about modelling this up in Swell? As far as I can see, I'll have to create 3 promotions for each category (spirits vs liqueurs) with a set minimum and maximum item quantity and a set discount? Is that the best way to do things, or is there a better way? Will setting up a promotion for X `quantity` of `category` work if the numbers are made up of different flavors within that category?

Thank you for your help.

**Comment by ericingram** [comment]
Check out product price rules to see if that fits your use case: https://www.swell.is/help/products/creating-products/product-pricing#price-rules

**Reply by Krislunde** [reply]
Hi @ericingram , thanks for your reply. That doesn't actually help. It would be perfect if we only wanted to offer bulk discounts on a single product, but we're hoping to offer a kind of "3 bottles of ANY (mixed) spirits will get a discount", as well as 6x and 12x.

Then a similar but different price level discount for bulk purchases of multiple liqueur products, which have a different pricing level. So essentially we are trying to model a bulk pricing discount on two different categories of products, not single products.

**Reply by ericingram** [reply, marked as answer]
Ah sorry I missed that point. A promotion as you described should work I think
